### PR TITLE
improve the syncMap.Iterate test to make it 100% reproducible

### PR DESCRIPTION
**Pull request by spacewander**

### CODE REVIEW

File: store/mocks/Store.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
```
Comment (asim) [inline store/mocks/Store.go]: Can you lower case this filename

Comment (vtolstov) [reply store/mocks/Store.go]: pass to mockery -case underscore so all output files will be lowercase
```
-case string
        name the mocked file using casing convention [camel, snake, underscore] (default "camel")
```

Comment (spacewander) [reply store/mocks/Store.go]: @asim @vtolstov 
Addressed in the new commit.

File: store/mocks/store.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
+
+package mocks
```
Comment (asim) [inline store/mocks/store.go]: Can this package be called "mock"?

Comment (vtolstov) [reply store/mocks/store.go]: hm, does some other projects uses mock and not mocks?
by the way:
```
 -outpkg string
       name of generated package (default "mocks")
 -output string
       directory to write mocks to (default "./mocks")
```
so this is easy to be setted to mock

Comment (spacewander) [reply store/mocks/store.go]: @asim @vtolstov 
Updated.

**Pull request merged**